pkg/config/config.go

```diff
@@ -92,6 +97,25 @@ func (cfg *MicroshiftConfig) KubeConfigPath(id KubeConfigID) string {
 	return filepath.Join(dataDir, "resources", string(id), "kubeconfig")
 }
 
+func (cfg *MicroshiftConfig) KubeConfigAdminPath(id string) string {
+	return filepath.Join(dataDir, "resources", string(KubeAdmin), id, "kubeconfig")
+}
+
+func getAllHostnames() ([]string, error) {
+	cmd := exec.Command("/bin/hostname", "-A")
+	var out bytes.Buffer
+	cmd.Stdout = &out
+	err := cmd.Run()
+	if err != nil {
+		return nil, fmt.Errorf("Error when executing 'hostname -A': %v", err)
+	}
+	outString := out.String()
+	outString = strings.Trim(outString[:len(outString)-1], " ")
+	names := strings.Split(outString, " ")
```

Comment:
```suggestion
	// remove any duplicate names
	names := strings.Split(outString, " ")
```

I was scratching my head for 10 seconds of why list->set->list

Reply:
That's it. The intention is to avoid duplicates in the SAN which would be confusing (and useless).